core/core-funcs.el

```diff
@@ -253,5 +253,11 @@ result, incrementing passed-tests and total-tests."
      (concat "Hidden Mode Line Mode enabled.  "
              "Use M-x hidden-mode-line-mode to make the mode-line appear."))))
 
+(defun spacemacs/recompile-elpa ()
+  "Recompile packages in elpa directory. Useful if you switch
+Emacs versions."
+  (interactive)
+  (byte-recompile-directory package-user-dir nil t))
```

Comment:
Yes, that makes sense, and you're right usually nil means don't change default behavior.

It shouldn't be a problem, but without knowing how the function works it's possible that `'force` gets evaluated twice which could lead to a void variable error or picking up the value of force in the context the function is evaluating in (who knows what that value could be).

Using a constant at least ensures that the number of times it is evaluated is irrelevant, which in principle is a little safer. Interestingly `"force"` and `[force]` are also constants